src/main/java/org/thief/repository/dao/Picture.java

```diff
@@ -0,0 +1,35 @@
+package org.thief.repository.dao;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "pictures")
+@Getter
+@Setter
+public class Picture {
```

Comment:
Please consider to think what is your hashcode & equals contract for entity.

Reply:
I don't see why we need it in the entity. What do you want to achieve?

Reply:
Please read https://vladmihalcea.com/hibernate-facts-equals-and-hashcode/ 
In some case we can ignore it, but it better to be aware.

Reply:
Thanks, i will take a look on it.

Reply:
Thanks again for sharing such info, I never read anything related to `equals` and `hashCode` in repository entities. I also looked at 2 other articles in that direction and it was really interesting to see that JPA Buddy recommendation is to use a single value `hashCode` (the one that has always the same values for all entities of the same class) for all entities if there is no unique key other than the primary key.